plugin.py

```diff
@@ -269,6 +271,40 @@ def transform_dcdc_converter_data(
             },
         )
 
+    # Typically, a SmartShunt in DC Energy Meter mode
+    def transform_dc_energy_meter_data(
+        self,
+        bl_device: BLEDevice,
+        cfg_device: ConfiguredDevice,
+        data: DcEnergyMeterData,
+        id_: str,
+    ) -> SignalKDeltaValues:
+        values = transformer(
+            f"electrical.batteries.{id_}",
```

Comment:
SignalK doesn't have all the schemas we need, but it does have some of them. We could select by mode:

* inverters (inverter mode)
* chargers (wind, shaft, fuel cell, regen, dc-dc, DC generator, AC, generic modes)
* alternators (alternator mode)
* solar (solar charger mode)

The venus plugin also uses these non-standard schemes, we could mirror them:
* `converters` (dc-dc convertors)
* `dcsource` (all dc sources)
* `dcload` (for all dc loads)

Reply:
oh that's a great point - I'll see if I can find time to do this today, we're probably not sailing...